site/content/en/docs/drivers/qemu.md

```diff
@@ -29,6 +29,33 @@ minikube start supports some qemu specific flags:
   * Note: while the flag should override the config, if the flag does not take effect try running `minikube delete`.
   * MacPorts: if you are installing [minikube](https://ports.macports.org/port/minikube/) and [qemu](https://ports.macports.org/port/qemu/) via MacPorts on a Mac with M1, use the following flag: `--qemu-firmware-path=/opt/local/share/qemu/edk2-aarch64-code.fd`
 
+## Networking
+
+The QEMU driver has two networking options, `user` & `socket_vmnet`.
+
+{{% tabs %}}
+{{% tab "user - limited functionality" %}}
+The `user` network is not a dedicated network, it doesn't support some networking commands such as `minikube service` and `minikube tunnel`.
```

Comment:
```suggestion
The `user` network is not a dedicated network, it doesn't support some networking commands such as `minikube service` and `minikube tunnel`, and its IP address is not reachable from the host.
```

Reply:
Thanks for the added information @AkihiroSuda